tests/puppeteer/campaigns/utils/files.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
   async doesFileExist(fileName, timeDelay = 5000, isPartialName = false, fileExtension = '') {
     let found = false;
     for (let i = 0; i <= timeDelay && !found; i += 100) {
```

Comment:
```suggestion
    for (let i = 0; i <= timeDelay && !found; i += 100) {
```

Maybe because of the 100 there ? Anyway.. this change nothing tbh